src/Tizen.Multimedia.Remoting/WebRTC/MediaSource.cs

```diff
@@ -552,6 +552,58 @@ public int EncoderBitrate
             }
         }
 
+        /// <summary>
+        /// Sets the RTP payload type of given <paramref name="codec"/>.
+        /// </summary>
+        /// <remarks>
+        /// <paramref name="payloadType"/> should be greater than or equal to 96 and less than or equal to 127.
+        /// </remarks>
+        /// <param name="codec">The transceiver codec.</param>
+        /// <param name="payloadType">The RTP payload type.</param>
+        /// <exception cref="InvalidOperationException">MediaSource is not attached yet.</exception>
+        /// <exception cref="ObjectDisposedException">The WebRTC has already been disposed.</exception>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///     <paramref name="payloadType"/> is less than 96 or greater than 127.
+        /// </exception>
+        /// <seealso cref="SupportedTransceiverCodecs"/>
+        /// <since_tizen> 12 </since_tizen>
+        public void SetPayloadType(TransceiverCodec codec, uint payloadType)
+        {
+            if (!SourceId.HasValue)
+            {
+                throw new InvalidOperationException("MediaSource is not attached yet. Call AddSource() first.");
+            }
+            if (payloadType < 96 || payloadType > 127)
+            {
+                throw new ArgumentOutOfRangeException(nameof(payloadType), payloadType, "Valid range : 96 <= payloadType <= 127");
+            }
+
+            NativeWebRTC.SetPaylodType(WebRtc.Handle, SourceId.Value, codec, payloadType).
+                ThrowIfFailed("Failed to set payload type");
+        }
+
+        /// <summary>
+        /// Gets the RTP payload type of given <paramref name="codec"/>.
+        /// </summary>
+        /// <param name="codec">The transceiver codec.</param>
+        /// <value>The RTP payload type.</value>
```

Comment:
`<value>` is used for properties, the return value of a method is documented with `<returns>`,
same for the other 2 new methods ( Get*Description() )
```suggestion
        /// <returns>The RTP payload type.</returns>
```

Reply:
I'll fix it.